src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    void returnInfo();
```

Comment:
The method name 'returnInfo' is not appropriate for an interface that is named 'AreaCalculator'. If the intent is to calculate the area, the method should be named accordingly, like 'calculateArea'.